src/elements/container/container/container.scss

```diff
@@ -38,4 +44,21 @@
   :host([expanded]) & {
     @include sbb.page-spacing-expanded;
   }
+
+  :host([data-slot-names~='image']) & {
+    position: relative;
+  }
+}
+
+::slotted([slot='image']) {
+  position: absolute;
+  inset: 0;
+  z-index: -1;
```

Comment:
This could cause problems with consumer stacking context. Would changing postion in DOM work? (And remove z-index here)

Reply:
Even changing the position in DOM there's always something (like titles) getting behind the image

Reply:
refactored a little bit